Userland/Libraries/LibPDF/Fonts/Type1Font.cpp

```diff
@@ -54,6 +54,10 @@ PDFErrorOr<void> Type1Font::initialize(Document* document, NonnullRefPtr<DictObj
         auto font = TRY(replacement_for(base_font_name().to_lowercase(), font_size));
 
         auto effective_encoding = encoding();
+        bool is_standard_14_font = base_font_name() == "Helvetica" || base_font_name() == "Helvetica-Bold" || base_font_name() == "Helvetica-Oblique" || base_font_name() == "Helvetica-BoldOblique"
+            || base_font_name() == "Times-Roman" || base_font_name() == "Times-Bold" || base_font_name() == "Times-Italic" || base_font_name() == "Times-BoldItalic"
+            || base_font_name() == "Courier" || base_font_name() == "Courier-Bold" || base_font_name() == "Courier-Oblique" || base_font_name() == "Courier-BoldOblique"
+            || base_font_name() == "Symbol" || base_font_name() == "ZapfDingbats";
```

Comment:
This is fine, but FYI you can save yourself some typing with `.is_one_of("Helvetia", "Helvetia-Bold", ...);`.

In the future, if this ever shows up in a profile, you might want to statically create these strings as fly strings, like we do in Userland/Libraries/LibWeb/HTML/AttributeNames.h. Then these comparisons will get the O(1) benefit.